samples/bluetooth/rssi_nrf53/prj.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_BT=y
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_LL_SOFTDEVICE_DEFAULT=y
```

Comment:
Is it necessary to add `CONFIG_BT_LL_SOFTDEVICE_DEFAULT=y` to the sample configuration?

Reply:
I thought it wasn't but it looks like it is (tested with #error during build)